### DISCUSSION TITLE
One of @remix-run/dev dependency has High vulnerabilities

### DISCUSSION THREAD

**Discussion by pahrizal** [remix-run/remix#2505] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/36581242/160230572-d2f11cc9-846a-48a7-b959-c7651ec2c0e6.png)

how to update this peer dependency?

**Comment by MichaelDeBoey** [comment, marked as answer]
I've created a PR to remove the unused `chalk-animation` dependency, which is causing this warning: https://github.com/remix-run/remix/pull/2547

**Comment by kentcdodds** [comment]
FWIW, there's no way that our use of that package will result in a problem. So it's a vulnerability that nobody can do anything with 🙃
